### PR TITLE
973_K_Closest_Points_to_Origin.py

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 973_K_Closest_Points_to_Origin.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Time: O(n)
+# Space: O(n)
```
Comment (ryujumoto) [inline 973_K_Closest_Points_to_Origin.py]: The time complexity and space complexity are wrong.
Look into the time complexity of each method of heapq.

File: 973_K_Closest_Points_to_Origin.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            element1, element2 = point
+            sum = -(element1 ** 2 + element2 ** 2)
+        if len(heap_list) < k:
+            heappush(heap_list, (sum, element1, element2))
+            
+        elif heap_list[0][0] < sum:
+            heapreplace(heap_list, (sum, element1, element2))
+            
+        return [[heap_element[1], heap_element[2]]
+                for heap_element in heap_list]
```
Comment (ryujumoto) [inline 973_K_Closest_Points_to_Origin.py]: Coｍbine these two lines into one line.

File: 973_K_Closest_Points_to_Origin.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+class Solution:
+     
+    def kClosest(self, points: List[List[int]], k: int) -> List[List[int]]:
+        heap_list = []
+        
+        for point in points:
+            element1, element2 = point
+            sum = -(element1 ** 2 + element2 ** 2)
+        if len(heap_list) < k:
+            heappush(heap_list, (sum, element1, element2))
```
Comment (ryujumoto) [inline 973_K_Closest_Points_to_Origin.py]: heapq.heappush

File: 973_K_Closest_Points_to_Origin.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            
+        elif heap_list[0][0] < sum:
+            heapreplace(heap_list, (sum, element1, element2))
```
Comment (ryujumoto) [inline 973_K_Closest_Points_to_Origin.py]: heapq.heapreplace

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I think I revised all points. Please check.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**